src/util/graphql.js

```diff
@@ -57,9 +57,28 @@ export const FETCH_EVENTS_QUERY = gql`
         email
       }
     }
+    getEventsReversed{
```

Comment:
These should be two separate queries. export cost fetch_events_query and then export const fetch_reversed_events_query